storage/src/nodestore.rs

```diff
@@ -1711,8 +1721,12 @@ impl<S: ReadableStorage> Iterator for FreeListIterator<'_, S> {
         };
 
         // update the next address to the next free block
```

Comment:
misplaced comment, with little value. The code it is attempting to comment is self documenting, so you can probably nuke this comment.

Bad comment examples:
```
// set x to 5
let x = 5;
// call the overflow method and return errors
overflow(x)?;
```